upup/models/cloudup/resources/addons/aws-load-balancer-controller.addons.k8s.io/k8s-1.9.yaml.template

```diff
@@ -749,8 +749,8 @@ spec:
       containers:
       - args:
         - --cluster-name={{ ClusterName }}
-        - --enable-waf=false
-        - --enable-wafv2=false
+        - --enable-waf={{ .AWSLoadBalancerController.EnableWAF }}
+        - --enable-wafv2={{ .AWSLoadBalancerController.EnableWAFv2 }}
```

Comment:
Not quite sure if these should apply to pre 1.19 clusters, but no strong opinions.